lib/faker/creature/bird.rb

```diff
@@ -0,0 +1,169 @@
+# frozen_string_literal: true
+
+module Faker
+  class Creature
+    class Bird < Base
+      flexible :bird
+
+      class << self
+        ##
+        # Produces a random common family name of a bird
+        #
+        # @return [String]
+        #
+        # @example
+        #   Faker::Creature::Bird.common_family_name #=> "Owls"
+        #
+        # @faker.version 2.16.0
```

Comment:
Can you update to `@faker.version next`?

```suggestion
        # @faker.version next
```

Reply:
> Can you update the similar parts of the other generators in this PR?

This is also the same :-)

Reply:
Derp, yup of course :laughing: